test/cadet_web/controllers/chat_controller_test.exs

```diff
@@ -76,6 +76,24 @@ defmodule CadetWeb.ChatControllerTest do
       end
     end
 
+    @tag authenticate: :student
+    @tag requires_setup: true
+    test "The content length is too long",
+         %{conn: conn, conversation_id: conversation_id} do
+      assert conversation_id != nil
+      max_message_length = ChatController.max_content_length()
+      message_exceed_length = String.duplicate("a", max_message_length + 1)
```

Comment:
Let's also add another testcase to test that `String.duplicate("a", max_message_length)` passes normally

Reply:
Added in later commit. However, in current setup the expected result is similar to the "conversation id does not match user id" case. Do you know how to fix this?